addons/stock/wizard/stock_picking_return.py

```diff
@@ -98,6 +143,16 @@ def _prepare_move_default_values(self, return_line, new_picking):
             'warehouse_id': self.picking_id.picking_type_id.warehouse_id.id,
             'origin_returned_move_id': return_line.move_id.id,
             'procure_method': 'make_to_stock',
+            'move_line_ids': [(0, 0, {
+                'picking_id': new_picking.id,
+                'product_id': return_line.product_id.id,
+                'product_uom_id': return_line.uom_id.id,
+                'location_id': return_line.move_id.location_dest_id.id,
+                'location_dest_id': self.location_id.id or return_line.move_id.location_id.id,
+                'lot_id': return_line.lot_id.id,
+                'package_id': return_line.package_id.id,
+                'qty_done': return_line.quantity,
+            })],
         }
```

Comment:
Means you're creating one move_line per move, and multiple moves are linked to a single origin_returned_move_id, there isn't any issues with this ?

Reply:
the multiple moves are merged in the return picking into a single one